debian/rules

```diff
@@ -35,9 +35,10 @@ SOURCE_DIR  = $(abspath $(PACKAGE_DIR)/../)
 #  embedded directly in the Python source...
 VERSION_UPSTREAM_DISTUTILS = $(shell sed -n 's/^version\s*=\s*"\(\S*\)"$$/\1/p' $(SOURCE_DIR)/pynicotine/utils.py)
 
+# Disable shares test, since shares are not saved for some reason...
```

Comment:
@kiplingw Do you have any idea why this fails in autopkgtest? The dbs/shelves for shares are seemingly empty after a scan.

Reply:
Seems to work when the tests run the second time, in debian/tests/control.

Reply:
No idea. Where is the log? Possibly a race condition.